utils.py

```diff
@@ -6673,8 +6673,42 @@ def isGoodForNERSC(self, no_step=False):
         #good = good &
         return good
 
+    def getTimeInfoForChain(self):
+        timeInfo = dict()
+        if 'Chain' in self.request['RequestType']:
+            base = self.request['RequestType'].replace('Chain','')
+            itask=1
+            carry_on = {}
+            while True:
```

Comment:
I'm always scared with these "While True", so you could replace it by something like:
```
for itask in range(1, self.request['TaskChain'] + 1):
```
this will yield the exact numbers of Tasks in the workflow.

Reply:
done